tests/unit/schemas/test_algorithm.py

```diff
@@ -19,15 +19,15 @@ def test_validate_blocking_keys(self):
                 blocking_keys=keys,
                 evaluators={},
                 rule="func:recordlinker.linking.matchers.eval_perfect_match",
-                cluster_ratio=0.5,
+                belongingness_ratio=[0.25, 0.5],
```

Comment:
Could we add a few tests to this file showing that the validators work as expected...lower has to be between 0 and 1, upper has to be between 0 and 1, lower has to be less than or equal to upper.